### DISCUSSION TITLE
Add world hint for music clue scrolls

### DISCUSSION THREAD

**Discussion by mjurksa** [runelite/runelite#15777] [Q&A] [answered]
Currently there is no world map hint for music clue scrolls. For consistency sake it would be a nice to have.

![221006_221016_java_Thin](https://user-images.githubusercontent.com/25037124/194409027-618b0c11-01ba-40cd-b27c-804e02591357.png)

**Comment by Nightfirecat** [comment, marked as answer]
Your PR to add this in https://github.com/runelite/runelite/pull/15778 was merged, this will be present in our next release.
